Jenkinsfile

```diff
@@ -258,7 +258,7 @@ pipeline {
                     fi
 
                     rm -fr out/results
-                    /opt/ort/bin/ort $LOG_LEVEL --stacktrace analyze $ALLOW_DYNAMIC_VERSIONS_OPTION $USE_CLEARLY_DEFINED_CURATIONS_OPTION -f JSON,YAML -i $PROJECT_DIR/source -o out/results/analyzer
+                    /opt/ort/bin/ort $LOG_LEVEL --stacktrace analyze $ALLOW_DYNAMIC_VERSIONS_OPTION $USE_CLEARLY_DEFINED_CURATIONS_OPTION -i $PROJECT_DIR/source -o out/results/analyzer
```

Comment:
Would it make to do the same for SpdxDocuments in this commit?

Reply:
I'm not sure what you mean, could you please clarify?

Reply:
As discussed via chat, we prefer to change the default SpdxDocument report format to only be YAML instead, to match the defaults of our CLI result files.